packages/wonder-blocks-announcer/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "@khanacademy/wonder-blocks-announcer",
+  "version": "0.0.1",
+  "design": "v1",
+  "description": "Live Region Announcer for Wonder Blocks.",
+  "main": "dist/index.js",
+  "module": "dist/es/index.js",
+  "source": "src/index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "types": "dist/index.d.ts",
+  "author": "",
+  "license": "MIT",
+  "publishConfig": {
+    "access": "public"
+  },
+  "dependencies": {
+    "@khanacademy/wonder-blocks-core": "^9.0.0"
+  },
+  "peerDependencies": {
+    "aphrodite": "^1.2.5",
+    "react": "18.2.0"
```

Comment:
We are using catalogs now btw! I had to update it in the [tabs package](https://github.com/Khan/wonder-blocks/pull/2514/files/e5c21ac6136c93d0dd360d1aa7d78cd961e4026f#r2012895205) too because the new package was still in a feature branch when https://github.com/Khan/wonder-blocks/pull/2505 was implemented!

Reply:
I will have to do that in a follow-up, unless there's a way around it on a feature branch!

Reply:
Ah good call! I made a new PR to get this change in: https://github.com/Khan/wonder-blocks/pull/2518

Reply:
Thanks for the quick review @beaesguerra!